src/main/java/uk/ac/ebi/embl/gff3tools/validation/ValidationRegistry.java

```diff
@@ -141,6 +144,13 @@ public List<ValidatorDescriptor> getValidations() {
     public List<ValidatorDescriptor> getFixs() {
         return cachedValidators.stream()
                 .filter(vd -> vd.clazz().isAnnotationPresent(Gff3Fix.class))
+                .filter(vd -> vd.method().isAnnotationPresent(FixMethod.class))
```

Comment:
If filtering FixMethod here, we can remove the respective filtering in executeFixs()